file-format-types.rdf

```diff
@@ -0,0 +1,202 @@
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="PageDescription">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PaginatedTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/PageDescription" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PageDescriptionGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="UnstructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:closeMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PlainTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/UnstructuredText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="StructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/StructuredText" />
+  </udfrs:GenreFacetType>
+  <!-- Structured text genres -->
+  <udfrs:GenreFacetType rdf:ID="SourceCode">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#SourceCode" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Markup">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/MarkupText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="HTML">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Markup" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#HTMLDocument" />
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300266021" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Presentation">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Presentation" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Presentation" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PresentationGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Spreadsheet">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
```

Comment:
I'm still a little suspicious of the OfficeDocument superclass: It seems
like it reifies a marketing strategy, doesn't it?

On Fri, Jun 5, 2015 at 11:56 AM, Esmé Cowles <notifications@github.com>
wrote:

> In file-format-types.rdf
> <https://github.com/duraspace/pcdm/pull/2#discussion_r31825169>:
>
> > +  <udfrs:GenreFacetType rdf:ID="HTML">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Markup" />
> > +    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#HTMLDocument" />
> > +    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300266021" />
> > +  </udfrs:GenreFacetType>
> > +  <udfrs:GenreFacetType rdf:ID="Presentation">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Document" />
> > +    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Presentation" />
> > +    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Presentation" />
> > +    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PresentationGenre" />
> > +  </udfrs:GenreFacetType>
> > +  <udfrs:GenreFacetType rdf:ID="Spreadsheet">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Document" />
>
> I think Spreadsheet could be a subclass of Dataset, as well as a subclass
> of an OfficeDocument type I introduced in barmintor#6
> <https://github.com/barmintor/pcdm/pull/6>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duraspace/pcdm/pull/2/files#r31825169>.
>